### PR TITLE
feat:add base EVM chain

### PR DESCRIPTION
**Pull request by skiv71**
## Description of the changes

Preparing to deploy to BASE

### CODE REVIEW

File: packages/currency/src/chains/evm/data/base.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { TokenMap } from '../../../types';
+
+import { supportedBaseERC20 } from 'currency/src/erc20/chains/base';
```
Comment (benjlevesque) [inline packages/currency/src/chains/evm/data/base.ts:3]: change to relative path import

**Review by benjlevesque — APPROVED** [review]
mind the PR title, it must be conventional (feat:...)

**Review by KolevDarko — APPROVED** [review]

**Review by leoslr — APPROVED** [review]

**Review by yomarion — APPROVED** [review]
+1 Benji's comments

File: packages/currency/src/chains/evm/data/base.ts (hunk 1 above)
Comment (skiv71) [reply packages/currency/src/chains/evm/data/base.ts]: done (was vscode auto complete, sorry)

**Pull request merged**